blocks/url-input/index.js

```diff
@@ -62,6 +62,17 @@ class UrlInput extends Component {
 			return;
 		}
 
+		// Disable suggestions for URLs
+		if ( /^https?:/.test( value ) ) {
```

Comment:
Is it worth merging this with the previous block?

Reply:
> Is it worth merging this with the previous block?

Yes, I think that's worthwhile: e171100cb